Fluid/Ast/TextSpanStatement.cs

```diff
@@ -1,16 +1,15 @@
-﻿using Parlot;
+﻿using Fluid.Utils;
+using Parlot;
 using System.Text.Encodings.Web;
-using Fluid.Utils;
 
 namespace Fluid.Ast
 {
     public sealed class TextSpanStatement : Statement
     {
-        private bool _isStripped;
-        private bool _isEmpty;
+        private bool _isBufferPrepared;
         private readonly object _synLock = new();
         private TextSpan _text;
-        private string _buffer;
+        internal string _preparedBuffer;
```

Comment:
Why `internal`?

Reply:
It's used internally in the branch it's coming from, will keep it this way for now.